src/Notifications/Characters/Slack/Killmail.php

```diff
@@ -68,12 +68,16 @@ public function via($notifiable)
     /**
      * Get the Slack representation of the notification.
      *
-     * @param $notifiable
+     * @param  $notifiable
      * @return SlackMessage
      */
     public function toSlack($notifiable)
     {
 
+        if (! parent::shouldProcessKillmail($notifiable)) {
```

Comment:
this should be done in routing section (`via`) - not the parser (`toX`) - since a notification is already ready to be sent at this stage